tasks/main.yml

```diff
@@ -0,0 +1,162 @@
+---
+- name: Install role requirements
+  import_tasks: requirements.yml
+  when: install_home_assistant_supervised_ansible_requirements|bool
+
+- name: Install OS package prereqs
+  apt:
+    name:
+      - systemd
+      - network-manager
+      - apparmor
+      - jq
+      - curl
+      - dbus
+    state: latest
+    update_cache: yes 
+    
+- name: Disable Modem Manager to prevent issues when using serial devices
+  service:
+    name: ModemManager
+    enabled: no
+    state: stopped
+
+- name: Create Docker config directory 
+  file:
+    path: /etc/docker 
+    state: directory
+
+- name: Set up docker config file
+  copy:
+    src: docker_daemon.json
+    dest: /etc/docker/daemon.json
+  notify:
```

Comment:
I haven't had much experience with handlers, but this is cool usage

Reply:
Yeah we probably could use them more. It's nice because it only triggers the handler if the task is changed, and if it gets triggered multiple times, it will only run the handler once.